README.md

```diff
@@ -1,13 +1,6 @@
 # Express API Project
 
-Replace this readme with your own information about your project. 
+This is my first API with several endpoints. It is connected to json file of Netflix movie database. This project was done in the framework of Technigo bootcamp for front-end development. Both types of path option and query search has been implemented.
 
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
 
-## The problem
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
-
-## View it live
-
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
```

Comment:
It would be very nice to have some instructions, how to use your API. Remember to make documentation on what endpoints and queries your user can use. 🤓